plugin-base/build.gradle.kts

```diff
@@ -80,12 +81,4 @@ val prepareProtocConfigVersions by tasks.registering {
 
 tasks.processResources.get().dependsOn(prepareProtocConfigVersions)
 
-//TODO:2021-07-22:alexander.yevsyukov: Turn to WARN and investigate duplicates.
-// see https://github.com/SpineEventEngine/base/issues/657
-val dupStrategy = DuplicatesStrategy.INCLUDE
-tasks.processTestResources.get().duplicatesStrategy = dupStrategy
-tasks.sourceJar.get().duplicatesStrategy = dupStrategy
-
-sourceSets.test {
-    resources.srcDir("$projectDir/src/test/resources")
-}
+apply<IncrementGuard>()
```

Comment:
So `IncremendGuard` is a plugin, right? Can we use it in the `plugins` section then?

Reply:
`plugins { }` doesn't support such plugins. `IncrementGuard` is declared as a Gradle plugin class in `buildSrc`. It doesn't have a plugin ID. While `plugins { }` work exclusively with plugin IDs (type-safe accessors such as `java-library` are just shortcuts for plugin IDs).